server/gc/safepoint_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gc
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tikv/pd/server/storage/endpoint"
+	"github.com/tikv/pd/server/storage/kv"
+)
+
+func newGCStorage() endpoint.GCSafePointStorage {
+	return endpoint.NewStorageEndpoint(kv.NewMemoryKV(), nil)
+}
+
+func TestGCSafePointUpdateSequentially(t *testing.T) {
+	gcSafePointManager := newGCSafePointManager(newGCStorage())
+	curSafePoint := uint64(0)
+	// update gc safePoint with asc value.
+	for id := 10; id < 20; id++ {
+		safePoint, err := gcSafePointManager.LoadGCSafePoint()
+		require.NoError(t, err)
+		require.Equal(t, safePoint, curSafePoint)
```

Comment:
This function's signature is `func Equal(expected interface{}, actual interface{})`, so the expected value should be placed on the first argument.